controllers/logging/kibana_controller.go

```diff
@@ -282,6 +283,14 @@ func (r *KibanaReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		GenericFunc: func(e event.GenericEvent) bool { return false },
 	}
 
+	// Watch for updates to the imagestream
+	isPred := predicate.Funcs{
+		UpdateFunc:  func(e event.UpdateEvent) bool { return true },
```

Comment:
I thought about restricting the watch to the ImageStream that is of interest to us, because there might be way more ImageStreams in customer clusters than are in our testing ones. But as we are only looking at the `openshift` namespace we should be fine.